src/3rdparty/kissfft/test/testkiss.py

```
@@ -4,158 +4,140 @@
     cmd += ','.join([str(d) for d in dims])
-    if doreal:
+    if do_real:
         cmd += ' -R '
 
-    print( cmd)
+    print(cmd)
 
-    from subprocess import Popen,PIPE
-    p = Popen(cmd,shell=True,stdin=PIPE,stdout=PIPE )
+    from subprocess import Popen, PIPE
+    p = Popen(cmd, shell=True, stdin=PIPE, stdout=PIPE)
```

Ruff (0.11.9)

`subprocess` call with `shell=True` identified, security issue

(S602)